.github/scripts/supported_python_versions.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+# 1.0 to 1.3 support python 3.7 to 3.10
+# 1.4 added support for python 3.11
+# 1.6 drops support for python 3.7
+# Note that Python 3.8 is manually added to the matrix so not included here
+
+minor_version="$1"
+if [[ ${minor_version} < 4 ]]
+then
+py_versions='["3.7", "3.8", "3.9", "3.10"]'
```

Comment:
I think all versions of core don't support 3.7 anymore

Reply:
That's only true of 1.6